htdocs/product/stock/movement_card.php

```diff
@@ -423,7 +431,7 @@
     $objectlabel = 'Movements';
     $permissiontoread = $user->rights->stock->lire;
     $permissiontodelete = $user->rights->stock->supprimer;
-    $uploaddir = $conf->stock->dir_output."/movement/";
+    $uploaddir = $conf->stock->movement->dir_output;
```

Comment:
It seems this var does not exist. We try to merge all 
dir_output and dir_temp into 1 dire depending on the $conf->module and then adding a suffix like 'movement'. So code will be ready for hashing directory trees in a future.
Old usage was better.

Reply:
The variable does exist: https://github.com/Dolibarr/dolibarr/blob/39bb69eb87e8d4b5d0ef3cdf59fa06084b668340/htdocs/core/class/conf.class.php#L437
It is part of this PR

Reply:
I changed this to be more compatible to other modules and their configurations. E.g. supplier orders and supplier invoices are configured the same way: https://github.com/Dolibarr/dolibarr/blob/39bb69eb87e8d4b5d0ef3cdf59fa06084b668340/htdocs/core/class/conf.class.php#L386